pkg/systemd/services/services.jsx

```diff
@@ -458,7 +458,10 @@
                                         this.seenPaths.add(unit_path);
 
                                         return this.getUnitByPath(unit_path);
-                                    }, ex => this.setState({ error: cockpit.format(_("Loading unit failed: $0"), ex.toString()), loadingUnits: false })));
+                                    }, ex => {
+                                        this.props.isLoading(false);
+                                        this.setState({ error: cockpit.format(_("Loading unit failed: $0"), ex.toString()) });
+                                    }));
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18005-20221207-164851-4022857c-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)